src/app/destiny2/d2-buckets.ts

```diff
@@ -12,39 +13,39 @@ import { D2ManifestDefinitions } from './d2-definitions';
 
 // A mapping from the bucket hash to DIM item types
 const bucketToTypeRaw = {
-  2465295065: 'Energy',
-  2689798304: 'UpgradePoint',
-  2689798305: 'StrangeCoin',
-  2689798308: 'Glimmer',
-  2689798309: 'Legendary Shards',
-  2689798310: 'Silver',
-  2689798311: 'Bright Dust',
-  3161908920: 'Messages',
-  3284755031: 'Class',
-  3313201758: 'Modifications',
-  3448274439: 'Helmet',
-  3551918588: 'Gauntlets',
-  3865314626: 'Materials',
-  4023194814: 'Ghost',
-  4274335291: 'Emblems',
-  4292445962: 'ClanBanners',
-  14239492: 'Chest',
-  20886954: 'Leg',
-  215593132: 'LostItems',
-  284967655: 'Ships',
-  375726501: 'Engrams',
-  953998645: 'Power',
-  1269569095: 'Auras',
-  1367666825: 'SpecialOrders',
-  1498876634: 'KineticSlot',
-  1585787867: 'ClassItem',
-  2025709351: 'Vehicle',
-  1469714392: 'Consumables',
-  138197802: 'General',
-  1107761855: 'Emotes',
-  1345459588: 'Pursuits',
-  1506418338: 'SeasonalArtifacts',
-  3683254069: 'Finishers',
+  [BucketHashes.EnergyWeapons]: 'Energy',
+  [BucketHashes.UpgradePoint]: 'UpgradePoint',
+  [BucketHashes.StrangeCoin]: 'StrangeCoin',
+  [BucketHashes.Glimmer]: 'Glimmer',
+  [BucketHashes.LegendaryShards]: 'Legendary Shards',
+  [BucketHashes.Silver]: 'Silver',
+  [BucketHashes.BrightDust]: 'Bright Dust',
+  [BucketHashes.Messages]: 'Messages',
+  [BucketHashes.Subclass]: 'Class',
+  [BucketHashes.Modifications]: 'Modifications',
+  [BucketHashes.Helmet]: 'Helmet',
+  [BucketHashes.Gauntlets]: 'Gauntlets',
+  [BucketHashes.Materials]: 'Materials',
+  [BucketHashes.Ghost]: 'Ghost',
+  [BucketHashes.Emblems]: 'Emblems',
+  [BucketHashes.ClanBanners]: 'ClanBanners',
+  [BucketHashes.ChestArmor]: 'Chest',
+  [BucketHashes.LegArmor]: 'Leg',
+  [BucketHashes.LostItems]: 'LostItems',
+  [BucketHashes.Ships]: 'Ships',
+  [BucketHashes.Engrams]: 'Engrams',
+  [BucketHashes.PowerWeapons]: 'Power',
+  [BucketHashes.Auras]: 'Auras',
+  [BucketHashes.SpecialOrders]: 'SpecialOrders',
+  [BucketHashes.KineticWeapons]: 'KineticSlot',
+  [BucketHashes.ClassArmor]: 'ClassItem',
+  [BucketHashes.Vehicle]: 'Vehicle',
+  [BucketHashes.Consumables]: 'Consumables',
+  [BucketHashes.General]: 'General',
+  [BucketHashes.Emotes_Invisible]: 'Emotes',
```

Comment:
pretty sure this should be Emotes_Equippable (individual emotes)
not Emotes_Invisible (this button)
![14aa589109bbd9e95df0757b23511b73 1](https://user-images.githubusercontent.com/68782081/151724220-ddd7c019-a032-4be7-b567-841510d5a1e9.jpg)

Reply:
but it's unclear if this is actually used anywhere, aside from a yet-unrealized ability to add emotes to loadouts